Snapshot management blog

kolchfa-aws opened the pull request:
### Description
Adds a blog on Snapshot Management.
 
### Issues Resolved
none

### Check List
- [x ] Commits are signed per the DCO using --signoff


By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

krisfreedain commented:
@kolchfa-aws @natebower just wanted to confirm this got through the editorial process before I publish. Please let me know - thanks

kolchfa-aws commented:
@krisfreedain - This is not ready to be merged yet. It needs to go through the tech review and editorial. I can put it in draft if it's easier

krisfreedain commented:
thanks for letting me know - I'll leave it until I hear from you again. thanks

natebower reviewed (commented):
@kolchfa-aws Please see my comments and let me know if you have any questions. Thanks!

kolchfa-aws commented on _posts/snapshot-management.md:
```diff
@@ -0,0 +1,145 @@
+categories:
+ - technical-post
+
+excerpt: "Snapshot Management is a way to automatically take snapshots of your cluster's indexes and state. This is useful for restoring indexes from a snapshot after a failure or migration. With Snapshot Management you can set a schedule for creating and deleting snapshots and a list of indexes to work on. You can also get notified of snapshot activity."
```
Made the excerpt the first couple of sentences of the body

kolchfa-aws commented:
@krisfreedain - I have incorporated editorial comments in the latest version of this blog.

krisfreedain commented:
thanks @kolchfa-aws - safe for me to push this one live?

kolchfa-aws commented:
@krisfreedain Sure. Let me know if you need anything from my side. Thanks!

krisfreedain commented on _posts/snapshot-management.md:
```diff
@@ -0,0 +1,145 @@
+
+The following image shows the creation workflow of the SM state machine.
+
+<img src="/assets/media/blog-images/2022-09-16/sm-state-machine.PNG" alt="SM State Machine" width="150" style="float: left; margin-right: 15px;"/>
```
please fix the path on this one - should be:
assets/media/blog-images/**2022-09-16-snapshot-management**/sm-state-machine.PNG

krisfreedain replied:
@kolchfa-aws - you will also need to create an author bio and image as part of this post. thanks

krisfreedain reviewed (changes requested):
@kolchfa-aws - two changes required 

- change for the image path needed
- also please rename your main file from snapshot-management.md to 2022-09-16-snapshot-management.md

kolchfa-aws commented:
@krisfreedain I fixed the image link and added author info. Also, I pushed the blog with a date in the name and without. Please feel free to delete or update if needed. Thanks!

kolchfa-aws commented:
@krisfreedain I need to collaborate with @setiah on this blog. Please don't merge until I make changes. Thanks.

krisfreedain commented:
> @krisfreedain I need to collaborate with @setiah on this blog. Please don't merge until I make changes. Thanks.

Sounds good. Please be sure to also complete your author bio file as well (kolchfa.markdown) - it's currently empty. Thanks

setiah commented:
@kolchfa-aws apologies for the long wait. I have revised a few aspects of the blog to make it factually correct and a bit more descriptive. I have also added a demo gif. You can find my changes in the PR i raised on your remote branch. Once you merge them they will automatically reflect in here. https://github.com/kolchfa-aws/project-website/pull/1

setiah commented:
> ### Description
> Adds a blog on Snapshot Management.
> 
> ### Issues Resolved
> none
> 
> ### Check List
> * [x ] Commits are signed per the DCO using --signoff
> 
> By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

pls add the issue link in the description - #987

natebower commented on _posts/2022-10-10-snapshot-management.md:
```diff
@@ -0,0 +1,158 @@
+
+**Scenario 1: Red Alert!** - Your cluster goes red because nodes went down, causing data loss. In this case, you can restore the red indexes from the most recent snapshot.
+
+**Scenario 2: Migration** - ** You need to migrate from one cluster to another. In this case, you can restore the indexes on your new cluster from a snapshot.
```
```suggestion
**Scenario 2: Migration** – ** You need to migrate from one cluster to another. In this case, you can restore the indexes on your new cluster from a snapshot.
```

natebower commented on _posts/2022-10-10-snapshot-management.md:
```diff
@@ -0,0 +1,158 @@
+
+**Scenario 2: Migration** - ** You need to migrate from one cluster to another. In this case, you can restore the indexes on your new cluster from a snapshot.
+
+**Scenario 3: Restore to a point in time view** - You need to revert back to a point-in-time view. You can restore a snapshot to go back to the previous state.
```
```suggestion
**Scenario 3: Restore to a point-in-time view** – You need to revert back to a point-in-time view. You can restore a snapshot to go back to the previous state.
```

natebower replied:
Should it be "a previous state"?

natebower commented on _posts/2022-10-10-snapshot-management.md:
```diff
@@ -0,0 +1,158 @@
+
+**Scenario 3: Restore to a point in time view** - You need to revert back to a point-in-time view. You can restore a snapshot to go back to the previous state.
+
+**Scenario 4: Save costs** - You need to save historical data for compliance, but you primarily use only last seven days of data. You can save all previous data in a snapshot and delete it from your cluster to save cost.
```
```suggestion
**Scenario 4: Save costs** – You need to save historical data for compliance, but you primarily use only the last seven days of data. You can save all previous data in a snapshot and delete it from your cluster to save costs.
```

natebower reviewed (commented):
@kolchfa-aws Review complete.

setiah commented on _posts/2022-10-10-snapshot-management.md:
```diff
@@ -0,0 +1,158 @@
+
+You can also set up your SM policy through OpenSearch Dashboards as follows:
+
+<img src="/assets/media/blog-images/2022-10-10-snapshot-management/Snapshot-management-demo.gif" alt="SM Policy Create Workflow" width="150" style="float: left; margin-right: 15px;"/>
```
@kolchfa-aws @krisfreedain how do we check this gif is rendering properly on the website?

krisfreedain replied:
@kolchfa-aws @setiah gif looks way too small:
![Screen Shot 2022-10-07 at 10 47 55 AM](https://user-images.githubusercontent.com/4873623/194619707-e0f6604b-e5a1-471e-b21e-a35de95b619d.png)


I recommend changing line 129 to:

`<img src="/assets/media/blog-images/2022-10-10-snapshot-management/Snapshot-management-demo.gif" alt="SM Policy Create Workflow"/>{: .img-fluid }`

kolchfa-aws replied:
Thank you, Kris! I will change that.

krisfreedain replied:
Excellent - thanks @kolchfa-aws - I'll push this live on Monday morning.
